p2p/transport/webrtc/udp_mux_conn.go

```diff
@@ -0,0 +1,246 @@
+package libp2pwebrtc
+
+import (
+	"encoding/binary"
+	"io"
+	"net"
+	"sync"
+	"time"
+
+	"github.com/pion/logging"
+	"github.com/pion/transport/packetio"
+)
+
+type udpMuxedConnParams struct {
```

Comment:
Please avoid configuration structs.

Reply:
This is an internal file from pion. I'll add a comment.